parsec/core/gui/create_org_widget.py

```diff
@@ -55,28 +50,13 @@ async def _do_create_org(
     device_label: DeviceLabel,
     backend_addr: BackendOrganizationBootstrapAddr,
 ) -> LocalDevice:
-    try:
-        new_device = await bootstrap_organization(
-            backend_addr, human_handle=human_handle, device_label=device_label
-        )
-        # The organization is brand new, of course there is no existing
-        # remote user manifest, hence our placeholder is non-speculative.
-        await user_storage_non_speculative_init(
-            data_base_dir=config.data_base_dir, device=new_device
-        )
-        return new_device
-    except InviteNotFoundError as exc:
-        raise JobResultError("invite-not-found", exc=exc)
-    except InviteAlreadyUsedError as exc:
-        raise JobResultError("invite-already-used", exc=exc)
-    except BackendConnectionRefused as exc:
-        raise JobResultError("connection-refused", exc=exc)
-    except BackendNotAvailable as exc:
-        raise JobResultError("connection-error", exc=exc)
-    except BackendOutOfBallparkError as exc:
-        raise JobResultError("out-of-ballpark", exc=exc)
-    except InviteError as exc:
-        raise JobResultError("invite-error", exc=exc)
+    new_device = await bootstrap_organization(
+        backend_addr, human_handle=human_handle, device_label=device_label
+    )
+    # The organization is brand new, of course there is no existing
+    # remote user manifest, hence our placeholder is non-speculative.
+    await user_storage_non_speculative_init(data_base_dir=config.data_base_dir, device=new_device)
+    return new_device
```

Comment:
I don't think `_do_create_org` make sense as function anymore, those two calls can be moved directly in the async event handler.

Reply:
fixed in 0eb544c49a90d6041bcc7f707f3ff8cfa59bbf48